### PR TITLE
Don't check autoplay setting when play is trigger by user gesture

### PR DESCRIPTION
**Pull request by darkdh**
fix https://github.com/brave/browser-laptop/issues/8897

Auditors: @bridiver, @bsclifton

### CODE REVIEW

**Comment by bridiver** [comment]
5.0 is the correct milestone for this?

**Comment by darkdh** [comment]
because 4.4.4 has been released so I chose the next available milestone.
It is for browser-laptop 0.20.x

File: patches/master_patch.patch [hunk 1]
```diff
@@ -1757,6 +1757,23 @@ index e54e2c0fcc8eac6fe637025c3c71f3d6834a4892..d817176d073d8a96fa76b944f6fb198d
...
  // Details about a redirection of a resource request.
+diff --git a/content/public/common/web_preferences.cc b/content/public/common/web_preferences.cc
+index 1e00ba53c4afcab6ccc747c255b032dccc373eb2..a61d28ff9ce5646259df66228acecead83e8b412 100644
+--- a/content/public/common/web_preferences.cc
++++ b/content/public/common/web_preferences.cc
+@@ -227,11 +227,8 @@ WebPreferences::WebPreferences()
+       presentation_receiver(false),
+       media_controls_enabled(true),
+       do_not_update_selection_on_mutating_selection_range(false),
+-#if defined(OS_ANDROID)
```
Comment (bridiver) [inline patches/master_patch.patch]: can you set these with `MUON_CHROMIUM_BUILD`?

Comment (darkdh) [inline patches/master_patch.patch]: ok, doing it now

Comment (darkdh) [inline patches/master_patch.patch]: changes pushed

File: patches/master_patch.patch [hunk 2]
```diff
@@ -1782,6 +1782,19 @@ index e54e2c0fcc8eac6fe637025c3c71f3d6834a4892..d817176d073d8a96fa76b944f6fb198d
...
+diff --git a/content/public/common/web_preferences.cc b/content/public/common/web_preferences.cc
+index 1e00ba53c4afcab6ccc747c255b032dccc373eb2..45693a865882770f19018eb030eb98d7a07ac382 100644
+--- a/content/public/common/web_preferences.cc
++++ b/content/public/common/web_preferences.cc
+@@ -227,7 +227,7 @@ WebPreferences::WebPreferences()
+       presentation_receiver(false),
+       media_controls_enabled(true),
+       do_not_update_selection_on_mutating_selection_range(false),
+-#if defined(OS_ANDROID)
++#if defined(OS_ANDROID) || defined(MUON_CHROMIUM_BUILD)
```
Comment (bridiver) [inline patches/master_patch.patch]: MUON_CHROMIUM_BUILD is not defined for our builds, it is only defined when building Chromium. Is that correct here?

Comment (bridiver) [inline patches/master_patch.patch]: the name is probably a little confusing, the MUON_ is just a prefix

Comment (darkdh) [inline patches/master_patch.patch]: Oh, I was confused by the prefix. And it has the opposite behavior

**Pull request merged**